proto/ibc/lightclients/tendermint/v1/tendermint.proto

```diff
@@ -57,6 +57,7 @@ message ClientState {
   // allow_update_after_misbehaviour is deprecated
   bool allow_update_after_misbehaviour = 11
       [deprecated = true, (gogoproto.moretags) = "yaml:\"allow_update_after_misbehaviour\""];
+  bytes absence_sentinel_value = 12 [(gogoproto.moretags) = "yaml:\"absence_sentinel_value\""];
```

Comment:
I find the naming could maybe be more descriptive. `nonexistent_key_sentinel_value`, or `sentinel_value_for_absence_key`. Don't have strong preferences, `absence` just feels like the wrong tense?

Reply:
How about `nonmembership_sentinel_value`?

Reply:
sounds smooth to me!